.github/workflows/dockerhub-master.yml

```diff
@@ -31,23 +31,6 @@ jobs:
           push: true
           build-args: COORDINATOR_FEATURES=metrics
 
-      - name: Apply kubeconfig
```

Comment:
the same steps exist in the `dockerhub-pr-with-parameters.yaml`, don't they need to be removed there as well?

Reply:
good catch! thanks